utils/protocol-types-generator/index.js

```diff
@@ -4,29 +4,37 @@ const fs = require('fs');
 const StreamZip = require('node-stream-zip');
 const vm = require('vm');
 const os = require('os');
+const util = require('util');
 
-async function generateChromiunProtocol(revision) {
+const existsAsync = path => fs.promises.access(path).then(() => true).catch(e => false);
+
+async function generateChromiumProtocol(executablePath) {
   const outputPath = path.join(__dirname, '..', '..', 'src', 'chromium', 'protocol.ts');
-  if (revision.local && fs.existsSync(outputPath))
+  if (!await existsAsync(executablePath))
```

Comment:
how can it not exist? This was previously guarding that a new revision that had been downloaded.